### ISSUE TITLE
WooCommerce Composite Products

### ISSUE THREAD

**Issue by taotsetung** [qTranslate-Team/qtranslate-x#265]
The problem:
When creating a woocommerce composite product I cannot save the component configuration.
After adding a few components and pressing save the loading animation is shown and even after 15 minutes nothing happend. When simply saving/updating the product while the loading animation is still shown it shows up as a simple product on the front page.

Initial Troubleshooting:
I enabled WP_DEBUG to have look at possible error messages, but nothing turned up (In fact three warnings turned up, but they are related to a widged I use and I get these warnings on all other pages too).
The error log is empty so that was no help either.
Javascript debug console also shows no errors at all. 
I also tried the whole thing in a local test environment with just the translation plugins, woocommerce and composite products with the same result.

Is this a known problem?
If not I would be happy to prepare a simple environment that shows the error, or any other help I can provide.

**Comment by johnclause** [comment]
No, I do not recall a report like this. Could you please, try the latest versions of everything from GitHub,  https://github.com/qTranslate-Team/qtranslate-x and https://github.com/qTranslate-Team/woocommerce-qtranslate-x, before digging any further?

**Comment by KVNHNF** [comment]
Dear John, 

First of all thank you for this great plugin! Qtranslate is awesome. 

Just want to stress out a new issue, I just bought Woocommerce composite product plugin, however even with the blue line displayed on the left, Qtranslate now fails to translate "Short product description" field in "product data". 

It works fine for variable products (I assume following your update as I have seen on a few threads you replied to) but not for composite products.

The format displayed is [eng]...[][fr]...[]

Many thanks for your help!

Best regards
